message SeesPuckStack {
  required bool viewLostId = 1; // robot specified by seenById can no 
                                // longer see this puckStack
  required fixed32 seenById = 2; // list of robots viewing
  optional float relx = 3;
  optional float rely = 4;
}

message PuckStack {
  //required fixed64 x = 1; //we don't use absolute positions anymore, so these should be obsolete.
  //required fixed64 y = 2; //Delete them when we're sure.
  required fixed32 stackSize = 1;
  repeated SeesPuckStack seespuckstack = 2;
}
